aspnetcore/release-notes/aspnetcore-6.0.md

```diff
@@ -0,0 +1,699 @@
+---
+title: What's new in ASP.NET Core 6.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 6.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/29/2021
+no-loc: [Home, Privacy, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, Kestrel]
+uid: aspnetcore-6.0
+---
+# What's new in ASP.NET Core 6.0
+
+This article highlights the most significant changes in ASP.NET Core 6.0 with links to relevant documentation.
+
+## ASP.NET Core MVC and Razor improvements
+
+## Minimal APIs
+
+Minimal APIs are architected to create HTTP APIs with minimal dependencies. They are ideal for microservices and apps that want to include only the minimum files, features, and dependencies in ASP.NET Core. For more information, see <xref:tutorials/min-web-api>.
```

Comment:
@Rick-Anderson  Can we also reference the Minimal API overview here? The info is hidden at the end of the Minimal API tutorial. I am afraid people might not be able to find it. It would be nice to give people a choice of reading the tutorial or the overview right away.